playground/src/state-view/state-view.scss

```diff
@@ -0,0 +1,20 @@
+@import 'base.scss';
+@import 'components/atoms/_text.scss';
+
+.state-view {
+  padding: 25px;
+
+  &-loader {
+    width: 45px;
+    height: 45px;
+    margin: 100px auto auto auto;
+    border: 2px solid lightgrey;
```

Comment:
We really need to get our shit together with those greys. Lightgrey is `rgb(211, 211, 211)` - our closest ones are `silver` with `rgb(204, 204, 204)` and `alto` with `rgb(223, 223, 223)`